inst/rmarkdown/templates/copernicus/skeleton/skeleton.Rmd

```diff
@@ -46,8 +46,6 @@ running:
 # This section is mandatory even if you declare that no competing interests are present.
 competinginterests: |
   The authors declare no competing interests.
-# OPTIONAL:
-algorithms: true
```

Comment:
This could be set to FALSE by default here so that it is not included unless opt-in ?

Is there a problem by doing that ?

Reply:
Same as above. But I see your point that there is a very slim chance that something gets somehow broken, but it does not make much sense to produce files that would not be accepted by the publisher.

Reply:
Update: this was probably not an issue in the past. However, according to what the publisher wrote to us, the way they are currently processing manuscripts is that the LaTeX code is translated into XML and then everything along the line is processed from there. It appears that their parser is not really forgiving. So probably it produced an error and before fixing that manually for every manuscript, they just send it back to the authors.

Reply:
yes I now see that it will still be used but the package needs to be installed already in the user system to avoid further error if `\begin{algorithm}` is used.

Reply:
Agreed!

Reply:
Maybe the most straightforward solution would be to mention that `algorithm` and/or `algorithmx` have to be installed when the environment is used, _in the part of the template where algorithms are shown?_

Reply:
Yes I think it should at least be mention in the part where it is used in the skeleton, maybe by showing a chunk with `include = FALSE` to install those packages so the at least the skeleton works. don't you think ?

Reply:
As written below, I guess we could circumvent this if checking and loading additional packages is allowed in the `copernicus_article.R`. @cderv the decision is all yours. I am ok with both solutions, but consider the integration in `copernicus_article.R` as more elegant.